explorer/explorerroutes.go

```diff
@@ -235,6 +235,54 @@ func (exp *explorerUI) Mempool(w http.ResponseWriter, r *http.Request) {
 	io.WriteString(w, str)
 }
 
+// Ticketpool is the page handler for the "/ticketpool" path
+func (exp *explorerUI) Ticketpool(w http.ResponseWriter, r *http.Request) {
+	chartData, groupedTickets, err := exp.explorerSource.TicketPoolVisualization(
+		dbtypes.ChartGroupingFromStr("all"),
+	)
+	if err != nil {
+		log.Errorf("Template execute failure: %v", err)
+		exp.StatusPage(w, defaultErrorCode, defaultErrorMessage, ErrorStatusType)
+		return
+	}
+	var mp = dbtypes.PoolTicketsData{}
+	exp.MempoolData.Lock()
+	var mpData = exp.MempoolData
+	exp.MempoolData.Unlock()
+
+	if len(mpData.Tickets) > 0 {
+		mp.Time = append(mp.Time, uint64(mpData.Tickets[0].Time))
+		mp.Price = append(mp.Price, mpData.Tickets[0].TotalOut)
+		mp.Mempool = append(mp.Mempool, uint64(len(mpData.Tickets)))
+	} else {
+		log.Warn("No tickets exists in the mempool")
+	}
+
+	str, err := exp.templates.execTemplateToString("ticketpool", struct {
+		Version     string
+		NetName     string
+		ChartData   []*dbtypes.PoolTicketsData
+		GroupedData *dbtypes.PoolTicketsData
+		Mempool     dbtypes.PoolTicketsData
```

Comment:
How come `GroupedData` is a pointer but not `Mempool`.  It's not a small struct.

Reply:
ok. I will fix that.